src/configurations/cancercomplexity/routesConfig.ts

```diff
@@ -487,19 +497,42 @@ const routes: GenericRoute[] = [
                 name: 'DetailsPage',
                 props: {
                   sql: datasetsSql,
-                  sqlOperator: 'LIKE',
-                  showMenu: false,
+                  sqlOperator: '=',
                   synapseConfigArray: [
                     {
-                      name: 'StandaloneQueryWrapper',
-                      title: 'Data',
-                      columnName: 'datasetAlias',
-                      tableSqlKeys: ['datasets'],
+                      name: 'CardContainerLogic',
+                      columnName: 'grantNumber',
+                      title: 'Related Grants',
+                      tableSqlKeys: ['grantNumber'],
                       props: {
-                        sql: filesSql,
                         sqlOperator: '=',
-                        rgbIndex: 0,
-                        title: 'Dataset Files',
+                        sql: grantsSql,
+                        ...grantsCardConfiguration,
+                        facetAliases,
+                      },
+                    },
+                    {
+                      name: 'CardContainerLogic',
+                      columnName: 'pubMedId',
+                      title: 'Related People',
+                      tableSqlKeys: ['publicationId'],
```

Comment:
We're getting closer to finishing the blockers for [PORTALS-2162](https://sagebionetworks.jira.com/browse/PORTALS-2162), so hopefully, if desired, this could become a list column in the coming weeks

Reply:
Awesome!!!! Thanks, @nickgros